README.md

```diff
@@ -10,6 +10,34 @@ on the provided pseudo-random seed.
 Each privileged application has its own sortition pool and is responsible for
 maintaining operator weights up-to-date.
 
+## Setup
+
+### Prerequisites
+
+* clone the repository
+* link:https://nodejs.org/en/[Node.js] v14.21.2
```

Comment:
Do we need to replace [Node.js] with a version? This link redirects to 404

Reply:
good catch! the other readmes are in adoc and this one is in md (which I should probably also fix)

Reply:
[ea4a41](https://github.com/keep-network/sortition-pools/pull/202/commits/ea4a415cf6812012154fd8806457a673198fd400)